dev/com.ibm.ws.security.mp.jwt/src/com/ibm/ws/security/mp/jwt/tai/MicroProfileJwtTAI.java

```diff
@@ -373,6 +370,25 @@ TAIResult getAndValidateMicroProfileJwt(HttpServletRequest request, HttpServletR
         return result;
     }
 
+    TAIResult getResultForMissingToken(HttpServletRequest request, HttpServletResponse response, String methodName) throws WebTrustAssociationFailedException {
```

Comment:
`String methodName` got brought in by my IDE when extracting this code to a separate method; it's not needed and should be removed in a followup.